README.Rmd

```diff
@@ -1,138 +1,98 @@
-  comment = "#>",
-  fig.path = "inst/assets/README_files/figure-markdown_github"
-)
-suppressWarnings(library(shiny.semantic))
-```
 
 ```{r, echo = FALSE, message = FALSE}
-source("readme_assets/ui_demo.R")
-
-create_example <- function(code, ...) {
-  unstyled <- deparse(substitute(code), width.cutoff = 20)
-  args <- as.list(unstyled)
-  args$sep <- "\n"
-  args <- as.list(styler::style_text(do.call(paste, args)))
-  args$sep <- "\n"
-  styled <- do.call(paste, args)
-  list(
-    htmlwidget = shiny.semantic::uirender(div(class="demo", code), ...),
-    code_demo = div(class="demo-code",
-      pre(code(class="r", styled))
-    )
-  )
-}
+suppressWarnings(library(shiny.semantic))
+suppressWarnings(library(shiny))
 ```
 
-<link href="http://fonts.googleapis.com/css?family=Maven+Pro:400,700|Inconsolata" rel="stylesheet" type="text/css">
-<link href='docs/style.css' rel='stylesheet' type='text/css'>
 
-`r params$repo_name`
+  `r params$repo_name`
 ===========
 
-<div class="subheader">Semantic UI wrapper for Shiny
+<img src="man/figures/hexsticker.png" align="right" alt="" width="120" />
+
+**Semantic UI wrapper for Shiny**
 
-</br>
+<!-- badges: start -->
+
+[![Travis build status](https://travis-ci.org/Appsilon/shiny.semantic.svg?branch=develop)](https://travis-ci.org/Appsilon/shiny.semantic) 
 
-[![Travis build status](https://travis-ci.org/Appsilon/shiny.semantic.svg?branch=develop)](https://travis-ci.org/Appsilon/shiny.semantic)
 [![codecov](https://codecov.io/gh/Appsilon/shiny.semantic/branch/master/graph/badge.svg)](https://codecov.io/gh/Appsilon/shiny.semantic)
-</div>
 
-<div class="section level2">
+[![cranlogs](https://cranlogs.r-pkg.org/badges/shiny.semantic)](https://CRAN.R-project.org/package=shiny.semantic)
+
+[![total](https://cranlogs.r-pkg.org/badges/grand-total/shiny.semantic)](https://www.rpackages.io/package/shiny.semantic)
+<!-- badges: end -->
+
+
 With this library it's easy to wrap Shiny with [Semantic UI components][semantic_repo_url]. Add a few simple lines of code and some CSS classes to give your UI a fresh, modern and highly interactive look.
 
 `master` branch contains the stable version. Use `develop` branch for latest features.
 
-
-<div class="ui stackable two column grid">
-<div class="column">
 <h2> Before </h2>
-<br>
- 
-```{r, echo = FALSE, message = FALSE, fig.height = 1, fig.width = 5.5}
-create_example(before_ui_demo(), width="100%", height="90%")$htmlwidget
-```
- 
-</div>
-<br>
-<div class="column">
+
+![Before](man/figures/ss_before.png)
+
 <h2> After </h2>
-<br>
- 
-```{r, echo = FALSE, message = FALSE, fig.height = 1, fig.width = 5.5}
-create_example(after_ui_demo(), width="100%", height="90%")$htmlwidget
-```
- 
-</div>
-</div>
-</div>
-
-<!-- #Basic tutorial article is available on [Appsilon Data Science blog](your_future_art_link). -->
-<!-- Live demo link below -->
-<!-- TODO Analogy to http://shiny.rstudio.com/gallery/widget-gallery.html -->
+
+![After](man/figures/ss_after.png)
```

Comment:
Not working:
![image](https://user-images.githubusercontent.com/7279715/79987469-be970b80-84ad-11ea-9832-8c031b046b74.png)